### DISCUSSION TITLE
Removal of _SCINTILLA table

### DISCUSSION THREAD

**Discussion by snoopy** [orbitalquark/textadept#554] [Q&A] [answered]
Quick question about the removal of the _SCINTILLA table.
It says in the commit to use `view` as replacement but for some things that doesn't seem possible. Instead, I used `buffer` where appropriate. Is that the right way to go?

For reference here are the changes I made in my fork of textredux https://github.com/snoopy/textredux/commit/186953c11ea6ba751a861fc484bfe4ed271abf26

**Comment by orbitalquark** [comment, marked as answer]
Yes, `buffer.EOL_LF` is the correct usage. The use of `view` instead of `_SCINTILLA` is mainly meant for the `new_*` functions. Constants have always been available in the `buffer` and `view` tables.
